src/main/java/it/pagopa/ecommerce/commons/documents/v2/info/TransactionInfo.java

```diff
@@ -0,0 +1,46 @@
+package it.pagopa.ecommerce.commons.documents.v2.info;
+
+import it.pagopa.ecommerce.commons.documents.v2.TransactionAuthorizationRequestData;
+import it.pagopa.ecommerce.commons.generated.server.model.TransactionStatusDto;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+/**
+ * Transaction info for dead-letter data
+ */
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Document
+public class TransactionInfo {
+
+    @NotNull
+    private String transactionId;
+
+    @NotNull
+    private String authorizationRequestId;
+
+    @NotNull
+    private TransactionStatusDto eCommerceStatus;
+
+    @NotNull
+    private TransactionAuthorizationRequestData.PaymentGateway gateway;
+
+    private List<String> paymentTokens;
```

Comment:
you can assume `notNull`

Reply:
done here 9ef26404c69fe0f32f00ad15b962c9d61009833f